streams/shared/src/main/scala/zio/stream/ZStreamChunk.scala

```diff
@@ -44,6 +44,35 @@ class ZStreamChunk[-R, +E, @specialized +A](val chunks: ZStream[R, E, Chunk[A]])
   final def collect[B](p: PartialFunction[A, B]): ZStreamChunk[R, E, B] =
     ZStreamChunk(self.chunks.map(chunk => chunk.collect(p)))
 
+  /**
+   * Drops the specified number of elements from this stream.
+   */
+  final def drop(n: Int): ZStreamChunk[R, E, A] =
```

Comment:
This made me change my mind on `StreamChunkEffect` - we probably do need that. There's no  way to implement this combinator  efficiently without an override.